style.css

```diff
@@ -231,141 +231,80 @@ body {
   margin-left: 49%;
 }
 
-/* popup desktop */
+.title-1 {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  margin-left: -5%;
+}
+
+/* popup mobile */
+
 .main {
+  width: 100vw;
+  height: 100%;
   position: fixed;
-  width: 2440px;
-  height: 1600px;
-  top: 900px;
-  background: #dfe0e2;
-  mix-blend-mode: normal;
-  margin-left: 1800px;
-  margin-top: -1000px;
-  z-index: 1;
+  top: 0;
+  background: #c1c7d0;
+  overflow-y: scroll !important;
 }
 
 .maincard {
-  grid-column: 8;
   display: flex;
   flex-direction: column;
   align-content: center;
   align-items: center;
-  border-radius: 8px;
-  margin-top: 204px;
-  margin-left: 960px;
-  width: 920px;
-  height: 1530px;
+  width: 100%;
   background: #fff;
-  border: 1px solid #ebecf0;
   box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
   z-index: 1;
+  height: 880px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 20px;
+  margin-bottom: 26px;
+  background-color: white;
+  border-radius: 16px;
 }
 
 .popup-image {
-  background-repeat: no-repeat;
-  width: 872px;
-  height: 616px;
-  left: 24px;
-  top: 78px;
-  border: 1px solid #dfe1e6;
-  border-radius: 8px;
+  width: 90%;
+  margin: 16px;
+  height: 230px;
+  display: block;
 }
 
```

Comment:
Good job so far, your popup is taking shape. There're however few tweaks we can do to improve our design 😉 
- [ ] [OPTIONAL] _My Recent Work_ section appears tilted to the left. Let's improve here by centering it.
- [ ] Popup on desktop appears far bigger and overflows the windows. Let's ensure that dimensions used matches that of the Figma template, and should always fits within the browser window.
- [ ] The Popup menu appears tilted to the left on desktop version, let's ensure it's centered.
- [ ] The popup should have a some level of transparency, as seen in the Figma template.
- [ ] Popup menu appears to overflow to the left on mobile screens. Let's ensure that the popup is contained within the mobile screen and properly padded accordingly as seen in the Figma template.

**Desktop popup:**
**Your desktop popup:**        |  **The original popup:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/226135154-6515b35a-3ef9-412a-9bc9-e3792d68dbd3.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/226135913-2d274918-ae17-456a-8456-347b6b73fe8c.png">

**Mobile popup:**
**Your Mobile popup:**        |  **The original popup:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/226135814-37bd10a9-af8e-46a3-ae05-86e4bcf3a5eb.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/226135437-140e5406-adbf-4962-b134-d5073620cc7a.png">